src/main/external-resources/renderers/Sony-BraviaXD.conf

```diff
@@ -0,0 +1,63 @@
+#------------------------------------------------------------------------------------
+# Profile for Sony Bravia XD75 / XD70 4K HDR avec Android TV (XD7505 / XD7005 Series)
+# See DefaultRenderer.conf for descriptions of all the available options.
+
+RendererName = Sony Bravia XD
+RendererIcon = Sony-BraviaX.png
+
+# ===================================================================================
+# This renderer has sent the following string/s:
+#
+# User-Agent: UPnP/1.0, X-AV-Physical-Unit-Info: pa="BRAVIA KD-65XD7505";
+" X-AV-Client-Info: av=5.0; cn="Sony Corporation"; mn="BRAVIA KD-65XD7505"; mv="3.0";
+# friendlyName=KD-65XD7505
+# udn=uuid:48454d48-01af-4343-ad82-70e21eb9dee1
+# manufacturer=Sony Corporation
+# modelName=KD-65XD7505
+# modelDescription=BRAVIA
+# manufacturerURL=http://www.sony.net/
+# ===================================================================================
+# https://www.sony.fr/electronics/televiseurs/xd7505-xd7005-series/specifications
+
```

Comment:
@Sami32 there could be some network setting restrictions which prevent the DLNA working like I have. This should be much safer way to recognise the renderer when only the DLNA is working.